Yubico.YubiKey/src/Yubico/YubiKey/Piv/PivSession.cs

```diff
@@ -215,6 +215,9 @@ public PivSession(IYubiKeyDevice yubiKey)
         /// <exception cref="ArgumentNullException">
         /// The <c>yubiKey</c> argument is null.
         /// </exception>
+        /// <exception cref="InvalidOperationException">
+        /// This exception can be thrown when unable to determine the management key type.
```

Comment:
By "can", do you mean that this exception is only sometimes thrown when the SDK is unable to determine the management key type? Or is it always thrown in this scenario?